### ISSUE TITLE
Feature Request: SceneShowStateChanged

### ISSUE THREAD

**Issue by bryce-seifert** [obsproject/obs-websocket#1037]
### Feature Request Type

RPC Event

### Requested Feature

Ability to have an event fire when the show state of source (scenes, groups, etc) that aren't inputs is changed.

### Requested Feature Usage Scenario

This would allow more accurate info about what sources are currently previewed. Right now, only inputs have accurate preview information thanks to the InputShowStateChanged event.

**Comment by ChristopheCVB** [comment]
Isn't [SceneItemEnableStateChanged](https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#sceneitemenablestatechanged) what you're looking for?

**Comment by bryce-seifert** [comment]
Not quite. I'm looking to get updates for the `videoShowing` parameter, similar to what's returned now with `InputShowStateChanged`

**Comment by ChristopheCVB** [comment]
Am I missing something or [InputShowStateChanged](https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#inputshowstatechanged) also fires for Nested Scenes and Groups

**Comment by bryce-seifert** [comment]
Correct, I'm looking to get info for scenes.

@tt2468 Said this will require some OBS-side changes to work
